server/routes/barcode/recipients/RecipientFormService.ts

```diff
@@ -37,24 +37,47 @@ export default class RecipientFormService {
     if (!req.session.recipients) {
       req.session.recipients = []
     }
+    const newRecipient = await this.toRecipient(recipientForm)
+
+    req.session.recipients.push(newRecipient)
+
+    req.session.recipientForm = {}
+  }
+
+  private async updateRecipient(req: Request, recipientForm: RecipientForm) {
```

Comment:
Does this method need to be async? Its not calling any external service is it? Its just updating the array of recipients in the session with the updated contact/recipient isnt it? 
(having said that it looks like the other methods in this class are also async, so perhaps theres no harm?)

Reply:
Yes it ends up calling prison register service to get the prison address - which is async